### PR TITLE
Added DB_SSL and DB_SSL_CA to helm chart

### PR DESCRIPTION
**Pull request by washcycle**

### CODE REVIEW

**Comment by mathuin** [comment]
Could you consider improving your pull request by adding an environment variable `DB_SSL_CA` that is based on something like `.Values.postgresql.sslCA` if that value is present?  That will help folks suffering from a self-signing certificate error which arises from node-postgres which is a dependency of a dependency.  The variable is already supported in the code at lines 64-77 of `core/db.js`.

**Comment by washcycle** [comment]
Excellent idea I will add that. Probably Monday, I'm off the grid this
weekend.

Cheers!

**Comment by washcycle** [comment]
Added the DB_SSL_CA option.

**Comment by washcycle** [comment]
Added documentation to the values.yml for this request.

File: dev/helm/README.md [hunk 1]
```diff
@@ -131,8 +131,6 @@ $ helm install --name my-release -f values.yaml .
 
 ## PostgresSQL
 
-By default, PostgreSQL is installed as part of the chart.
-
```
Comment (NGPixel) [inline dev/helm/README.md:135]: Why is this removed?

File: dev/helm/values.yaml [hunk 2]
```diff
@@ -80,9 +80,25 @@ postgresql:
...
   ##
   enabled: true
+  ## ssl enforce SSL communication with PostgresSQL
+  ## Default to false
+  ##  
+  # ssl: false
+  ## ca Certificate of Authority
+  ## Default to empty, point to location of CA
+  ##  
+  # ca: "path to ca"
```
Comment (NGPixel) [inline dev/helm/values.yaml:90]: Should be added to the Configuration section of the README as well.

File: dev/helm/values.yaml [hunk 3]
```diff
@@ -80,9 +80,25 @@ postgresql:
...
+  ##    
+  # postgresqlHost: postgres
+  ## postgresqlPort port for postgres 
+  ## Default to 5432
+  ##    
+  # postgresqlPort: 5432
   ## PostgreSQL fullname Override
   ## Default to wiki-postgresql unless fullname override is set for Chart
-  ##
+  ##
```
Comment (NGPixel) [inline dev/helm/values.yaml:101]: Unnecessary trailing spaces

**Review by NGPixel — CHANGES_REQUESTED** [review]

File: dev/helm/README.md (hunk 1 above)
Comment (washcycle) [reply dev/helm/README.md:135]: Merge mistake. Good catch I'll add that back.

File: dev/helm/values.yaml (hunk 2 above)
Comment (washcycle) [reply dev/helm/values.yaml]: Added notes in the REAME on using the ssl/ca settings.

File: dev/helm/values.yaml (hunk 3 above)
Comment (washcycle) [reply dev/helm/values.yaml]: I think I fixed this one... and found a few other trailing spaces in some other spots.

File: dev/helm/README.md [hunk 4]
```diff
@@ -137,6 +137,8 @@ By default, PostgreSQL is installed as part of the chart.
 
 To use an external PostgreSQL server, set `postgresql.enabled` to `false` and then set `postgresql.postgresqlHost` and `postgresql.postgresqlPassword`. The other options (`postgresql.postgresqlDatabase`, `postgresql.postgresqlUser` and `postgresql.postgresqlPort`) may also want changing from their default values.
 
+To use an SSL connection you can set `postgres.ssl` to `true` and if needed the path to a Certificate of Authority can be set using `postgres:ca` to `/path/to/ca`. Default `postgres.ssl` value is `false`.
```
Comment (NGPixel) [inline dev/helm/README.md:140]: You have a typo `postgres:ca`.

Also add the 2 options to the configuration table above.

**Review by NGPixel — CHANGES_REQUESTED** [review]

File: dev/helm/README.md (hunk 4 above)
Comment (washcycle) [reply dev/helm/README.md]: done

File: dev/helm/README.md [hunk 5]
```diff
@@ -137,6 +139,8 @@ By default, PostgreSQL is installed as part of the chart.
 
 To use an external PostgreSQL server, set `postgresql.enabled` to `false` and then set `postgresql.postgresqlHost` and `postgresql.postgresqlPassword`. The other options (`postgresql.postgresqlDatabase`, `postgresql.postgresqlUser` and `postgresql.postgresqlPort`) may also want changing from their default values.
 
+To use an SSL connection you can set `postgresql.ssl` to `true` and if needed the path to a Certificate of Authority can be set using `postgresql:ca` to `/path/to/ca`. Default `postgresql.ssl` value is `false`.
```
Comment (NGPixel) [inline dev/helm/README.md:142]: You still have the `postgres:ca` typo, should be `postgres.ca`

**Review by NGPixel — CHANGES_REQUESTED** [review]

File: dev/helm/README.md (hunk 5 above)
Comment (washcycle) [reply dev/helm/README.md:142]: oh man seeing things, my bad

**Review by NGPixel — APPROVED** [review]
